src/App.js

```diff
@@ -0,0 +1,24 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Booklist from './components/booklist';
+import Categories from './components/categories';
+import Navbar from './components/navbar';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <div className="App">
+        <Navbar />
+        <main>
+
+          <Routes>
+            <Route path="/" element={<Booklist />} />
+            <Route path="/categories" element={<Categories />} />
+          </Routes>
```

Comment:
- [ ] Kindly note that you have to create another component that will hold all your book-related components. That's the component that will be imported here, not the `booklist` component.
So, your `book` component will be imported into the `booklist` component to display individual book items in a list. then the `booklist` and the `input` component will be imported into the new component you will create which will then be imported into the main component, the `App.js`.